yarn.lock

```diff
@@ -5136,6 +5169,18 @@ quill@2.0.0-dev.4:
     parchment "2.0.0-dev.2"
     quill-delta "4.2.1"
 
+quill@^1.2.2:
+  version "1.3.7"
+  resolved "https://registry.yarnpkg.com/quill/-/quill-1.3.7.tgz#da5b2f3a2c470e932340cdbf3668c9f21f9286e8"
+  integrity sha512-hG/DVzh/TiknWtE6QmWAF/pxoZKYxfe3J/d/+ShUWkDvvkZQVTPeVmUJVu1uE6DDooC4fWTiCLh84ul89oNz5g==
+  dependencies:
+    clone "^2.1.1"
+    deep-equal "^1.0.1"
+    eventemitter3 "^2.0.3"
+    extend "^3.0.2"
+    parchment "^1.1.4"
+    quill-delta "^3.6.2"
+
```

Comment:
By mistake? 
we are using [quill@2.0.0-dev.4](https://github.com/frappe/frappe/pull/10820/files#diff-8ee2343978836a779dc9f8d6b794c3b2L5127)

Please update yarn.lock

Reply:
ok

Reply:
@surajshetty3416 it is a dependency of `quill-image-resize@^3.0.9`

```
quill-image-resize@^3.0.9:
  version "3.0.9"
  resolved "https://registry.yarnpkg.com/quill-image-resize/-/quill-image-resize-3.0.9.tgz#5677fb6257560bff951153ddbdb836758e49f804"
  integrity sha512-5Dk0nixhbFsCwSWtPU9qqqtfM2gURfaP+pbBhQvAoMJoF4p99xbAibfAI3gsZJkbWUodoK2iAPf1V5oTSpv9ww==
  dependencies:
    lodash "^4.17.4"
    quill "^1.2.2"
    raw-loader "^0.5.1"
```